static/app/components/profiling/flamegraph/flamegraph.tsx

```diff
@@ -554,7 +554,7 @@ function Flamegraph(): ReactElement {
         strategy,
         flamegraphView.configView,
         new Rect(frame.start, frame.depth, frame.end - frame.start, 1)
-      );
+      ).transformRect(flamegraphView.configSpaceTransform);
```

Comment:
might be something we should add as a helper function or even change `computeConfigViewWithStrategy` to take the `flamegraphView` directly

Reply:
I thought about adding directly to computeConfigViewWithStrategy but I dont like that it creates the hidden dependency on the transform. I'm 50/50 on it tbh